src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala

```diff
@@ -3,13 +3,32 @@ package com.itechart.ny_accidents.constants
 import com.typesafe.config.ConfigFactory
 
 object Configuration {
-  private val appConf = ConfigFactory.load("app.conf")
-  private val cacheConf = ConfigFactory.load("conf/cache.conf")
+  private lazy val appConf = ConfigFactory.load("conf/app.conf")
```

Comment:
Could you clarify why didn't you use default name application.conf  for main config file?
https://github.com/lightbend/config#standard-behavior

Reply:
Fixed

Reply:
One more place requires fix

Reply:
+